src/electionguard_cli/cli_models/e2e_inputs.py

```diff
@@ -14,15 +14,18 @@ def __init__(
         guardians: List[Guardian],
         manifest: Manifest,
         ballots: List[PlaintextBallot],
+        spoil_id: str,
```

Comment:
As a general question, any number of ballots could be spoiled. Shouldn't this just be checking the status of the ballot itself?

Reply:
Per our discussion I have created the following tickets: 
- https://github.com/microsoft/electionguard-python/issues/581 for reading in plaintext ballots from directories
- https://github.com/microsoft/electionguard-python/issues/585 for reading in ciphertext ballots from directories
- https://github.com/microsoft/electionguard-python/issues/586 for reading in submitted ballots from a file

Please let me know if I captured the requirements correctly or not.